tests/cpython-tests/Makefile

```diff
@@ -23,20 +24,29 @@ ext2fs: appdir
 	$(MYST) mkext2 appdir ext2fs
 
 clean:
-	rm -fr appdir ext2fs @test_101*
+	rm -fr appdir ext2fs @test_101* hostfs
 
 tests:
-	$(RUNTEST) $(MYST_EXEC) $(OPTS) ext2fs /cpython/python -m test -f tests.passed -q
-	$(RUNTEST) $(MYST_EXEC) $(OPTS) ext2fs /cpython/python -m test -f tests.passed.1 -q
+	$(RUNTEST) ./exec.sh
 
-run-single:
-	$(RUNTEST) $(MYST_EXEC) $(OPTS) ext2fs /cpython/python -m test $(TC) -v
+#########################
+##	Dev targets
+#########################
+hostfs:
+	cp -r appdir/ hostfs/
 
-run-str:
-	$(MYST_EXEC) $(OPTS) ext2fs /cpython/python -c "$(PROG_STR)"
+run-list: $(FS)
+	$(MYST_EXEC) $(OPTS) $(FS) /cpython/python -m test -f $(TESTFILE) --timeout 30 -q
 
-pdb:
-	$(RUNTEST) $(MYST_EXEC) $(OPTS) ext2fs /cpython/python -m pdb /cpython/Lib/test/test_grammar.py -v 
+
+run-single: $(FS)
```

Comment:
The command to run a single test from other test suites are like: `make one TEST=<path-to-test>`.

Reply:
Renamed.